Site1.Master

```diff
@@ -59,7 +65,7 @@
                             <asp:LinkButton class="nav-link" ID="LinkButton2" runat="server" OnClick="LinkButton2_Click">Sign Up</asp:LinkButton>
                         </li>
                         <li class="nav-item active">
-                            <asp:LinkButton class="nav-link" ID="LinkButton3" runat="server" Visible="False">Logout</asp:LinkButton>
+                            <asp:LinkButton class="nav-link" ID="LinkButton3" runat="server" Visible="False" OnClick="LinkButton3_Click">Logout</asp:LinkButton>
```

Comment:
ID is still showing as **LinkButton3**. 
```suggestion
                            <asp:LinkButton class="nav-link" ID="lbSignUp" runat="server" Visible="False" OnClick="lbSignUp_Click">Logout</asp:LinkButton>
```

Bad naming convention. Please give this a normal name.